editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2274,6 +2275,50 @@ function fillMissingDataFromAncestors(mergedMetadata: ElementInstanceMetadataMap
     }
   })
 
+  const nullsInConditional = Object.keys(workingElements).filter((p) => {
+    const element = workingElements[p]
+    const isNull =
+      element?.element != null && isLeft(element.element) && element.element.value == 'null'
+    if (!isNull) {
+      return false
+    }
+    const parentElement = workingElements[EP.toString(EP.parentPath(EP.fromString(p)))]
+    return MetadataUtils.isConditionalFromMetadata(parentElement)
+  })
```

Comment:
Down the line it would be nice to take these "passes" out into separate functions, so that each new addition is forced into its own context, rather than interleaved with the existing stuff